library/core/src/bool.rs

```diff
@@ -2,7 +2,7 @@
 
 #[lang = "bool"]
 impl bool {
-    /// Returns `Some(t)` if the `bool` is `true`, or `None` otherwise.
+    /// Returns <code>[Some]\(t)</code> if the `bool` is [`true`](keyword.true.html), or [`None`] otherwise.
```

Comment:
* Linking true seems a little excessive
* I don't think we need to link Some or None, those are similarly to true basic vocabulary terms